### PR TITLE
CodeofConduct.md: Remove in favor of org-wide CoC

### PR DESCRIPTION
**Pull request by astronouth7303**
Use https://github.com/ppb/.github/blob/master/CODE_OF_CONDUCT.md instead.

### CODE REVIEW

**Comment by astronouth7303** [comment]
I'm not seeing this linked from anywhere?

**Comment by pathunstrom** [comment]
Considering we've discussed moving to netlify for hosting, would this go against their requirements for an open source license?

Specific reference:
> Features a Code of Conduct at the top level directory of the project repository or prominently in the documentation (with a link in the navigation, footer, or homepage).

I guess resolvable by duplicating and/or linking to the Contributors Covenant in the docs.

**Comment by astronouth7303** [comment]
GitHub used to show the CoC at the top of the project, next to the license?

If it's not any longer, we should link to it in all the places.

**Comment by pathunstrom** [comment]
> I'm not seeing this linked from anywhere?

https://github.com/ppb/pursuedpybear/blob/master/CONTRIBUTING.md

It's linked to in Contributing.

**Comment by pathunstrom** [comment]
> GitHub used to show the CoC at the top of the project, next to the license?
> 
> If it's not any longer, we should link to it in all the places.

I think it's shown when you open a new pull request. In the same place Contributing.MD is shown.

**Comment by astronouth7303** [comment]
I opened a request with GitHub support.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,5 +126,5 @@ end you contribute to `ppb` by being part of its community.
 [issues]: https://github.com/ppb/pursuedpybear/issues "PPB Issues"
 [new contributor]: https://github.com/ppb/pursuedpybear/labels/new%20contributor "Issues labeld New Contributor"
 [projects]: https://github.com/orgs/ppb/projects "PPB Projects"
-[readme]: https://github.com/pathunstrom/pursuedpybear/blob/master/README.md "PPB README"
```
Comment (pathunstrom) [inline CONTRIBUTING.md]: Wow, that one was never updated? Impressive.

**Comment by pathunstrom** [comment]
bors try

**Pull request merged**